### PR TITLE
Adds the Ultra brick

### PR DESCRIPTION
**Pull request by nicbn**
## About The Pull Request

Adds the brick from Ultra, a parody for a real life brick which costs hundreds of dolars.

## Why It's Good For The Game

image

image

(ALL SPRITERS: LEAVE THIS PR NOW)

image

## Changelog
:cl:
add: Brick
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
make it break windows

**Comment by CRITAWAKETS** [comment]
not even the omega brick smh

**Comment by LemonInTheDark** [comment]
that commit history is pain

**Comment by Qustinnus** [comment]
https://www.youtube.com/watch?v=FgyonkrEWik

if it was a ref to this I would've thought the pr was pretty good

**Comment by D4C-420** [comment]
Feel like a solid hit to the head with this should do some ULTRA DAMAGE or low stamina damage at the least.

**Comment by nicbn** [comment]
I'll make it so that if you throw at windows it bounces backs at you

File: code/game/objects/items/miscellaneous.dm [hunk 1]
```diff
@@ -314,6 +314,22 @@
...
 	to_chat(M, "<span class='hear'>You hear something crackle from the beacon for a moment before a voice speaks. \"Please stand by for a message from S.E.L.F. Message as follows: <b>Item request received. Your package has been transported, use the autosurgeon supplied to apply the upgrade.</b> Message ends.\"</span>")
 
+/obj/item/ultrabrick
+	name = "Ultra brick"
+	desc = "A brick made by Ultra."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "expensive_brick"
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 10
+	throwforce = 5
```
Comment (ATH1909) [inline code/game/objects/items/miscellaneous.dm:324]: the throwforce should be much higher than the normal force

**Comment by cacogen** [comment]
Sprite's too big, those damage values don't make sense and if it doesn't have some sort of interesting but not overpowered unique functionality then what's the point in adding it? It'll be bought a couple of times for the novelty when it's still new and then never touched again. Yes, yes, the Supreme brick is also useless, well done, well done. At least give it a high sell value on the cargo shuttle or something.

>I'll make it so that if you throw at windows it bounces backs at you

Ok. and this should give the user brain damage. Or just buying it should.

File: code/game/objects/items/miscellaneous.dm (hunk 1 above)
Comment (cacogen) [reply code/game/objects/items/miscellaneous.dm:324]: the throwforce should also be useless

**Comment by nicbn** [comment]
I'll let the idea age a bit to become fine wine

**Comment by nicbn** [comment]
I'm probably going to bundle this change with the addition of regular bricks

**Comment by Tlaltecuhtli** [comment]
will you add clay in lavaland for homemade bricks?

**Comment by nicbn** [comment]
idk, would it be a good idea?

**Pull request closed without merging**